mmrotate/models/dense_heads/rotated_atss_head.py

```diff
@@ -1,15 +1,21 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-import torch
-from mmdet.core import images_to_levels, multi_apply, unmap
+from typing import List, Optional
 
-from mmrotate.core import obb2hbb, rotated_anchor_inside_flags
-from ..builder import ROTATED_HEADS
-from .rotated_retina_head import RotatedRetinaHead
+import torch
+from mmdet.models import RetinaHead
+from mmdet.models.task_modules.prior_generators import anchor_inside_flags
+from mmdet.models.utils import images_to_levels, multi_apply, unmap
+from mmdet.structures.bbox import BaseBoxes
+from mmdet.utils import InstanceList, OptInstanceList
+from mmengine.data import InstanceData
+from torch import Tensor
+
+from mmrotate.registry import MODELS
 from .utils import get_num_level_anchors_inside
 
 
-@ROTATED_HEADS.register_module()
-class RotatedATSSHead(RotatedRetinaHead):
+@MODELS.register_module()
+class RotatedATSSHead(RetinaHead):
```

Comment:
Why here inherit from RetinaHead not ATSSHead?

Reply:
Because this Rotated ATSS is not true ATSS, it don't have the centersness loss.

Reply:
See PR #496